index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Medical core</title>
+</head>
+<body>
+    <div class="medical-header__container">
+        <header class="medical-header">
+            <h1 class="medical-header__logo"> <a class="medical-header__anchor" href="">Medical Core</a></h1>
+            <ul class="medical-header__list">
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Home</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Product</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Pricing</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Contact</a></li>
+            </ul>
+    
+            <ul class="medical-header__list">
+                <li class="medical-header__item--login"><a class="medical-header__anchor--login" href="">Login</a></li>
+                <li class="medical-header__item--signup"><a class="medical-header__anchor--signup" href="">Join Us</a></li>
+            </ul>
+        </header>
+
+
+        <div class="medical-banner">
+            <h4 class="medical-banner__header">Welcome</h4>
+            <p class="medical-banner__paragraph--1">A Great Place to Receive Care </p>
+            <p class="medical-banner__paragraph--2">Get your best looking smile now!</p>
+    
+            <ul class="medical-banner__list">
+                <li class="medical-banner__list__item medical-banner__list__item--join"> Join Us</li>
+                <li class="medical-banner__list__item medical-banner__list__item--learn">Learn More</li>
```

Comment:
`medical-banner__list__item` `medical-banner__list__item--join` `medical-banner__list__item--join` `medical-banner__list__item--learn` this is not bem. it was a old way of naming nested element. follow this link [here](https://scalablecss.com/bem-nesting-grandchild-elements/)